README.md

```diff
@@ -1 +1,225 @@
+
+# THIERRY NOUMBUH (AXCELL EVANS)
+
+  <h1><b>Awesome-project</b></h1>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- 
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+<!-- PROJECT DESCRIPTION -->
+
+<!-- this is my new blog I created with HTML, CSS -->
+
+# portfolio
+This project is Awesome-projects microvers which will be used for learning in the microverse. No complex codind is required for this exercise. The goal is to master all of the tools and best practices learned in previous steps. They will be used in all Microverse projects and most likely in future jobs as well, so it is important to know!.
+1- HTML
+2- css
+3- linters
+4- javascript
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">HTML</a></li>
+    <li><a href="https://reactjs.org/">CSS</a></li>
+    <li><a href="https://reactjs.org/">Javascript</a></li>
+
+  </ul>
+</details>
+
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+- added index.HTML
+- added app.CSS
+- added script.js
+- added About.html
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+# LIVE DEMO 
+
+- [] coming soon
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+1- web browser
+2- git-syn
+3- code editor
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Run this command:
+  cd my-folder
+  git clone https://github.com/axcell-0/awesome-project.git
+
+### Install
+
+Install this project with:
+
+
+Run this command:
+
+...sh
+  cd my-project
+  npm install
+  ...
+
+### Usage
+
+To run the project, execute the following command:
+
+Example command:
+open index.html using the live server
+
+
+### Run tests
+
+To run tests, run the following command:
+
+[]comming soon
+
+### Deployment
+
+You can deploy this project using:
+
+ [] coming soon
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## Author <a name="author"></a>
+
+## 👤 THIERRY NOUMBUH (AXCELL EVANS)
+## 👤 YOUNG GAIUS
+## 👤 ZUWAIRA SADIQ (ZUWAIRANAJMA)
```

Comment:
- [ ] [OPITONAL] Additionally, it might be helpful to include the social media handles of your team members, so that people can easily get in touch with them if they have questions or want to learn more about your team. This can be a great way to increase visibility and engagement for your project. Thank you!